main/src/ca/uwaterloo/flix/language/phase/jvm/GenExpression.scala

```diff
@@ -691,6 +691,12 @@ object GenExpression {
         mv.visitMethodInsn(INVOKEINTERFACE, interfaceType.jvmName.toInternalName, interfaceType.RestrictFieldMethod.name,
           MethodDescriptor.mkDescriptor(BackendObjType.String.toTpe)(interfaceType.toTpe).toDescriptor, true)
 
+      case AtomicOp.VariantIs(label) => ??? // TODO: Ext-Variants
```

Comment:
Gasp, adding TODO's

Reply:
Yes, but what else? I cannot do everything in one PR.

Reply:
I think the boilerplate is response is that we don't do `TODO`, they should be tickets instead

Reply:
That is only for TODOs that will be left over for a long time.

Reply:
hmm I'll keep that in my back pocket